### ISSUE TITLE
Enhancement: 

### ISSUE THREAD

**Issue by nicksavov** [digitalbiblesociety/browserbible-3#78]
First off, I love BibleWebApp. Great job with it!

It would be cool if you could add a read-along feature to the audio Bibles and text. Checkout https://github.com/westonruter/html5-audio-read-along for a proof of concept.

**Comment by nicksavov** [comment]
Wow! Looks like you already have an auto-scroll option with the audio. All that's missing is something to highlight the text. This is particularly useful for young readers and those learning in a second language.

**Comment by johndyer** [comment]
Yeah, this would be a great feature. It's just a matter of somehow generating the data that Weston did in a reliable way. The audio aligner he used doesn't always work even wiht english.
